generated/Destinations.ts

```diff
@@ -0,0 +1,927 @@
+/*
+ GENERATED - DO NOT EDIT
+ This file is generated from the templates/constants.ts.template file.
+ */
+
+export class Destinations {
```

Comment:
Exporting all the constants as part of a class would not allow tree shaking them during build time and will bloat the SDK as this includes non-device mode destinations as well.

We should explore exporting each of the names as a standalone string.

Reply:
Please suggest desire code to be generated so that I can update template